azurelinuxagent/common/osutil/default.py

```diff
@@ -80,15 +80,18 @@ def get_accept_tcp_rule(wait, command, destination):
 
 
 def get_firewall_accept_command(wait, command, destination, owner_uid):
-    return AddFirewallRules.get_iptables_accept_command(wait, command, destination, owner_uid)
+    return AddFirewallRules.get_accept_command(command, destination, owner_uid, wait=wait)
```

Comment:
Man, this being used in lot of places and especially in all test cases. I don't know the reason behind this. I'm afraid to change this might break test cases or need lot of changes.

Reply:
Don't be afraid of change (especially for tests, you will need to get your hands dirtier sooner or later haha). It shouldn't be too much work :)

Reply:
challenge accepted, lol. I have addressed it now :)